README.md

```diff
@@ -31,7 +31,7 @@ runMain com.gu.newsletterlistcleanse.TestGetCleanseList
 
 ```sbtshell
 eval System.setProperty("Stage", "CODE")
-runMain com.gu.newsletterlistcleanse.UpdateBrazeUsersLambda
+runMain com.gu.newsletterlistcleanse.TestUpdateBrazeUsers
```

Comment:
If we can drop this change from this PR, I can move it into  https://github.com/guardian/newsletters-list-cleanse/pull/6/ where it fits more in terms of the general code changes.

Reply:
sounds good, feel free to report the change and I'll rebase